internal/pkg/mapper/sharedcode/helper/helper.go

```diff
@@ -77,75 +95,24 @@ func (h *SharedCodeHelper) GetTargetComponentId(sharedCodeConfig *model.Config)
 	return model.ComponentId(componentId), nil
 }
 
-func (h *SharedCodeHelper) GetSharedCodePath(object model.Object) (*model.Config, string, error) {
-	// Shared code is used by transformation
-	ok, err := h.IsTransformation(object.Key())
-	if err != nil || !ok {
-		return nil, "", err
-	}
-	transformation := object.(*model.Config)
-
-	// Path is stored in configuration
-	sharedCodePathRaw, found := transformation.Content.Get(model.SharedCodePathContentKey)
-	if !found {
-		return nil, "", nil
-	}
-
-	// Path must be string
-	sharedCodePath, ok := sharedCodePathRaw.(string)
-	if !ok {
-		return nil, "", fmt.Errorf(`key "%s" must be string, found %T, in %s`, model.SharedCodePathContentKey, sharedCodePathRaw, object.Desc())
-	}
-	return transformation, sharedCodePath, nil
-}
-
-func (h *SharedCodeHelper) GetSharedCodeKey(object model.Object) (*model.Config, model.Key, error) {
-	// Shared code is used by transformation
-	ok, err := h.IsTransformation(object.Key())
-	if err != nil || !ok {
-		return nil, nil, err
-	}
-	transformation := object.(*model.Config)
-
-	// ID is stored in configuration
-	sharedCodeIdRaw, found := transformation.Content.Get(model.SharedCodeIdContentKey)
-	if !found {
-		return nil, nil, nil
-	}
-
-	// ID must be string
-	sharedCodeConfigId, ok := sharedCodeIdRaw.(string)
-	if !ok {
-		return nil, nil, fmt.Errorf(`key "%s" must be string, found %T, in %s`, model.SharedCodeIdContentKey, sharedCodeIdRaw, object.Desc())
-	}
-
-	// Id -> key
-	sharedCodeKey := model.ConfigKey{
-		BranchId:    transformation.BranchId, // same branch
-		ComponentId: model.SharedCodeComponentId,
-		Id:          model.ConfigId(sharedCodeConfigId),
-	}
-	return transformation, sharedCodeKey, nil
-}
-
 func (h *SharedCodeHelper) GetSharedCodeByPath(branchKey model.BranchKey, path string) (*model.ConfigState, error) {
 	// Get branch
 	branch, found := h.state.Get(branchKey)
 	if !found {
-		return nil, fmt.Errorf(`%s not found`, branchKey.Desc())
+		return nil, fmt.Errorf(`missing %s`, branchKey.Desc())
 	}
 
 	// Get key by path
 	path = filesystem.Join(branch.Path(), path)
 	keyRaw, found := h.naming.FindByPath(path)
 	if !found {
-		return nil, fmt.Errorf(`shared code "%s" not found`, path)
+		return nil, fmt.Errorf(`missing shared code "%s"`, path)
 	}
 
 	// Is config?
 	key, ok := keyRaw.(model.ConfigKey)
 	if !ok {
-		return nil, fmt.Errorf(`path "%s" it not config`, path)
+		return nil, fmt.Errorf(`path "%s" it not shared code config`, path)
```

Comment:
Sice to není v upravovaným kódu 🙂 ale tady je asi typo, nemá to být "is not"?

Reply:
dik, upravim